### PR TITLE
Add Laravel Docs Plugin

### PR DESCRIPTION
**Pull request by Henriquex25**
A simple plugin to search and access Laravel documentation

[Repository](https://github.com/Henriquex25/laravel-doc-flow-launcher-plugin)

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1947,5 +1947,19 @@
...
+        "Description": "Laravel Documentation Search Engine",
+        "Author": "Henrique Miranda",
+        "Version": "1.0.0",
+        "Language": "TypeScript",
+        "Website": "https://github.com/Henriquex25/laravel-doc-flow-launcher-plugin",
+        "UrlDownload": "https://github.com/Henriquex25/laravel-doc-flow-launcher-plugin/releases/download/v1.0.0/laravel-doc-flow-launcher-plugin.zip",
+        "UrlSourceCode": "https://github.com/Henriquex25/laravel-doc-flow-launcher-plugin",
+        "IcoPath": "https://github.com/Henriquex25/laravel-doc-flow-launcher-plugin/blob/main/assets/laravel.png?raw=true",
+        "DateAdded": "2023-11-07T06:56:46Z",
+        "LatestReleaseDate": "2023-11-07T06:56:46Z"
```
Comment (jjw24) [inline plugins.json:1963]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/Henriquex25/laravel-doc-flow-launcher-plugin@main/assets/laravel.png"
```

**Comment by jjw24** [comment]
No need to add `DateAdded` and `LatestReleaseDate` fields, they will be auto populated. I updated the icon url to use mirror link.

**Review by jjw24 — CHANGES_REQUESTED** [review]
Hey the executable path does not exist: ./dist/main.js
Do you mean ./src/main.ts in the plugin.json file?

Plugin currently failing to load with the latest release.

**Comment by Henriquex25** [comment]
> Hey the executable path does not exist: ./dist/main.js Do you mean ./src/main.ts in the plugin.json file?
> 
> Plugin currently failing to load with the latest release.

I used the [flow-launcher-plugin-template-node](https://github.com/Joehoel/flow-launcher-plugin-template-node) repository as a base.

the javascript file is being built in the path ./dist/main.js

**Comment by jjw24** [comment]
At the moment flow is not able to find the executable file indicated in plugin.json. let me know once you have sorted this issue I will re-test

**Comment by jjw24** [comment]
> > Hey the executable path does not exist: ./dist/main.js Do you mean ./src/main.ts in the plugin.json file?
> > Plugin currently failing to load with the latest release.
> 
> I used the [flow-launcher-plugin-template-node](https://github.com/Joehoel/flow-launcher-plugin-template-node) repository as a base.
> 
> the javascript file is being built in the path ./dist/main.js

the downloaded plugin zip does not contain  ./dist/main.js

**Comment by taooceros** [comment]
let's wait #319

**Comment by jjw24** [comment]
The issue here is not related to that PR though

**Comment by jjw24** [comment]
Closing, please resubmit when ready.

**Pull request closed without merging**